ecommerce/frontend/src/components/layout/Layout_common.less

```diff
@@ -0,0 +1,34 @@
+@import '../../common.less';
+
+.header-logo {
+    float: left;
+    margin-left: 10px;
+    margin-right: 10px;
+    font-size: 2em;
+}
+
+.header {
+    padding-left: 0px;
+    padding-right: 0px;
+    position: 'fixed';
+    z-index: 1;
+    width: 100%;
+    background: @Eggplant;
```

Comment:
Instead of Eggplant, we may choose softer colors for header, and 1-bit softer color for category bar. Just an idea, we can discuss later.

Reply:
Also, if we show login and signup buttons at header, they need to have the same style as whole header.

Reply:
I could not make a comment on favicon.ico, I do it here : 
Putting logo on tab section is a good idea, however, since the icon is pretty dark. It doesn't look good. 
Later, we can open issue and redesign the coloring of our icon, for this case. 
I think, it is a good idea for us to have 2 different colored versions of our logo. 
Black background + @Champagne font
@Champagne font + Black font

Reply:
Yes I also noticed that the favicon looks too dark as it is right now. It's good that you have changed it to getflix logo rather than react default icon, but we should definitely state this to the group in the next meeting.

Reply:
> Also, if we show login and signup buttons at the header, they need to have the same style as the whole header.

do you mean the entire bar should be the same color? so like either all white or all that color?



> I could not make a comment on favicon.ico, I do it here :
> Putting logo on tab section is a good idea, however, since the icon is pretty dark. It doesn't look good.
> Later, we can open issue and redesign the coloring of our icon, for this case.
> I think, it is a good idea for us to have 2 different colored versions of our logo.
> Black background + Champagne font
> Champagne font + Black font



> Yes I also noticed that the favicon looks too dark as it is right now. It's good that you have changed it to getflix logo rather than react default icon, but we should definitely state this to the group in the next meeting.

I agree with you guys, I preferred to leave color selection and style changes for later so that we all agree on it. This pull request was mainly for structural purposes and having something on the screen.